tests/instron_stress_rig.py

```diff
@@ -622,17 +632,16 @@ def test_WHEN_the_quarter_counter_is_armed_THEN_the_number_of_quarts_never_excee
         cycles = 5
         self.ca.set_pv_value(quart_prefixed("CYCLE:SP"), cycles)
         self.ca.assert_that_pv_is(quart_prefixed("SP"), cycles*4)
-        self.ca.set_pv_value(quart_prefixed("ARM"), 1)
-        self.ca.assert_that_pv_is(quart_prefixed("STATUS"), "Armed")
+        self.ca.set_pv_value(wave_prefixed("START"), 1)
         while self.ca.get_pv_value(quart_prefixed("STATUS")) == "Armed":
-            self.assertLessEqual(4*float(self.ca.get_pv_value("QUART")), cycles)
-        self.ca.assert_that_pv_is(quart_prefixed("STATUS"), "Off")
+            self.assertLessEqual(float(self.ca.get_pv_value("QUART")/4.0), cycles)
+        self.ca.assert_that_pv_is(quart_prefixed("STATUS"), "Tripped")
```

Comment:
This test was previously incorrect but passing because of a bug in the emulator.